pkg/registry/apis/provisioning/register.go

```diff
@@ -237,6 +242,38 @@ func (b *ProvisioningAPIBuilder) afterDelete(obj runtime.Object, opts *metav1.De
 		return
 	}
 }
+func (b *ProvisioningAPIBuilder) Mutate(ctx context.Context, a admission.Attributes, o admission.ObjectInterfaces) error {
```

Comment:
Can we get a type assertion of this implementing `builder.APIGroupMutation`?

Reply:
i am not following. What do you mean?

Reply:
Have a `var _ builder.APIGroupMutation = (*ProvisioningAPIBuilder)(nil)` somewhere